.github/workflows/cli.yml

```diff
@@ -0,0 +1,66 @@
+name: run CLI
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  basic-usage:
+    name: Build on node ${{ matrix.node_version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, macOS-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node_version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node_version }}
+    - name: building
+      run: |
+        npm install
+        npm run build
+    - name: create workspace
+      run: mkdir -p myworkspace
+    - name: test client basic info
+      working-directory: ./myworkspace
+      run: |
+        ../packages/cli/dist/bin/index.js -v
+        ../packages/cli/dist/bin/index.js --version
+        ../packages/cli/dist/bin/index.js -h
+        ../packages/cli/dist/bin/index.js --help
+        ../packages/cli/dist/bin/index.js init --help
+        ../packages/cli/dist/bin/index.js run --help
+        ../packages/cli/dist/bin/index.js log --help
+        ../packages/cli/dist/bin/index.js board --help
+        ../packages/cli/dist/bin/index.js plugin-dev --help
+        ../packages/cli/dist/bin/index.js dataset --help
+        ../packages/cli/dist/bin/index.js bip --help
+        ../packages/cli/dist/bin/index.js serve --help
+    - name: test client basic operate init
+      working-directory: ./myworkspace
+      run: |
+        mkdir init1
+        cd init1
+        ../packages/cli/dist/bin/index.js init
+        cd ../
+        mkdir init2
+        cd init2
+        ../packages/cli/dist/bin/index.js init --beta
+        cd ../
+        mkdir init3
+        cd init3
+        ../packages/cli/dist/bin/index.js init --tuna
```

Comment:
We need a script to check if these init commands are working correctly.

Reply:
I will try:)